lib/webdriveragent.js

```diff
@@ -371,7 +393,17 @@ class WebDriverAgent {
     }
 
     this.setupProxies(sessionId);
-    const status = await this.getStatus();
+    let status;
+    try {
+      status = await this.getStatus(this.wdaLaunchTimeout);
+    } catch (err) {
+      this.log.debug(`Failed to get the status. Error ${err.message}`);
```

Comment:
Not sure if we actually need a log there as we anyway know it may only happen on timeout